.agents/skills/get-reviews/fetch_reviews.sh

```
@@ -0,0 +1,29 @@
+              author { login }
+              body
+              url
+              createdAt
+            }
+          }
+        }
+      }
+    }
+  }
+}' -F owner=fedify-dev -F repo=fedify -F number=$PR_NUMBER \
+> 'plans/$PR_NUMBER/fetched/$CURRENT_TIMESTAMP_MMDDHHMM.json'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)